### DISCUSSION TITLE
December|Community Manager Monthly Report

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#63] [Show and tell]
# Overview

December witnessed our first community round table. Good start！DCDAO’s constitution version 1.0 is done.
<img width="434" alt="image" src="https://github.com/dcdao/discussion/assets/109792098/48a31988-86c7-4632-9443-daa34864a925">

# Details

### Christmas Campaign with the other partner

Description: Christmas Campaign 

LINK:https://x.com/Official_DCDAO/status/1737317951215374525?s=20

Data Review:1127 participants;Twitter follower 1K(300 new followers); discord members 2600+

### Voice Call

Description: 1st Community Round Table 

https://x.com/Official_DCDAO/status/1734179616066044020?s=20

Data Review: https://x.com/Official_DCDAO/status/1736001050442600711?s=20

## Community Management

DCDAO& Darwinia Quiz Game will be a regular event in Discord server. 

11 members joined the event on average in this month.

https://discord.gg/KwQHF6sv

### DCDAO’s Constitution

Link：https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xd750140eccee3434f65feb156c985f6425ed884d56aa2a295c189d5e76fe9dc5

# January Plan
<img width="489" alt="image" src="https://github.com/dcdao/discussion/assets/109792098/8fe1e1e2-9e29-4737-9949-2560e463eb57">

**Comment by hackfisher** [comment]
Hi Cindy, I have some questions about twitter operation:

https://github.com/orgs/dcdao/discussions/64
